styles.css

```diff
@@ -234,5 +234,103 @@ body {
 
   .separa {
     width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
```

Comment:
- [ ] I think that what the reviewer was trying to say is that you reset your global spacings by defining a global selector `*` and setting the `margin` and `padding` to 0. This will make sure that every item has no spacings by default so you can easily style each one of them explicitly (also getting rid of the annoying white margin surrounding your page). 🤓 🔧 


💡 At the top of your stylesheet, write this rule:
```css
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box; /* This last one is optional, but I highly recommend using it */
}

```

You can read more about the [CSS box model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model), and the [box-sizing property](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing). 🤓 🔧